src/Microsoft.Identity.Client/Instance/AdfsOpenIdConfigurationEndpointManager.cs

```diff
@@ -124,24 +105,7 @@ protected override bool ExistsInValidatedAuthorityCache(string userPrincipalName
                 }
             }
 
-            return GetDefaultOpenIdConfigurationEndpoint();
-        }
-
-        protected override string GetDefaultOpenIdConfigurationEndpoint()
-        {
-            return CanonicalAuthority + ".well-known/openid-configuration";
-        }
-
-        protected override void AddToValidatedAuthorities(string userPrincipalName)
-        {
-            var authorityInstance = this;
-            if (ServiceBundle.ValidatedAuthoritiesCache.TryGetValue(CanonicalAuthority, out Authority authority))
-            {
-                authorityInstance = (AdfsAuthority)authority;
-            }
-
-            authorityInstance._validForDomainsList.Add(GetDomainFromUpn(userPrincipalName));
-            ServiceBundle.ValidatedAuthoritiesCache.TryAddValue(CanonicalAuthority, authorityInstance);
+            return authorityInfo.CanonicalAuthority + ".well-known/openid-configuration";
```

Comment:
>".well-known/openid-configuration" [](start = 54, length = 34)

const in helper, seems to be used in a number of places? perhaps something that combines (authorityInfo.CanonicalAuthority)